### ISSUE TITLE
BUG

### ISSUE THREAD

**Issue by RocXelL** [v1s1t0r1sh3r3/airgeddon#387]
#### What is your airgeddon version?

10.21 

#### What is your Linux O.S. and which version?

Kali Linux 2020.3

#### Which is the chipset of your wireless card?

RTL8188EUS 

#### Describe the issue and the steps to reproduce it

Ok so what happens is that im trying to do a evil twin attack and basically it does create the evil twin AP and I can capture the handshake.
But when doing the last part of the attack which is deauthing the "victim" ap and creating a fake AP to phish the password; what happens is that the window which is supposed to deauth the victim ap just stays stuck in "Waiting for beacon frame".
And I tried manually deauthing with aireplay-ng but the fake AP dissapears.
So basically it's either I deauth or I make a fake AP.
I would appreciate the help!

**Comment by OscarAkaElvis** [comment]
I think I already answered you in the other issue. I mean... check about compatible cards: 

https://github.com/v1s1t0r1sh3r3/airgeddon/wiki/Cards%20and%20Chipsets

Your card is having known problems. The TL-WN722N adapter has a v1 which works fine... yours is v2 and it is known that it has some problems. Usually, chipsets RTL8x have problems as you can see them in the "blacklist" of our compatibility section at wiki.

I suggest to you to buy a nice adapter. And if you want more information, join us on Discord. The invitation link is: https://discord.gg/sQ9dgt9
